server/index.js

```diff
@@ -0,0 +1,7 @@
+const cookieParser = require("cookie-parser");
```

Comment:
why import it here and in app.js file?

Reply:
there's no need to call it here, I forgot to remove it